examples/mnist_with_visdom.py

```diff
@@ -116,7 +133,12 @@ def validation_inference_function(batch):
         correct = pred.eq(target.data.view_as(pred)).sum()
         return loss, correct
 
-    trainer = Trainer(train_loader, training_update_function, val_loader, validation_inference_function)
+    def get_validation_inference_handler(validatation_data):
```

Comment:
As this could be replaced by the `Validate` handler, I think it's worth adding a comment here saying that in most cases it can be used, but that the user is free to create its own handler

Reply:
Yes, I didn't saw it before. I can change like [here](https://github.com/pytorch/ignite/pull/45#issuecomment-356774652).